use-cases/using-Intel-SGX-and-kata.md

```diff
@@ -0,0 +1,68 @@
+# Kata Containers with SGX
+
+- [Install Host kernel with SGX support](#install-host-kernel-with-sgx-support)
+- [Install Guest kernel with SGX support](#install-guest-kernel-with-sgx-support)
+- [Run Kata Containers with SGX enabled](#run-kata-containers-with-sgx-enabled)
+
+Intel® Software Guard Extensions (SGX) is a set of instructions that increases the security
+of applications code and data, giving them more protections from disclosure or modification.
+At the time of writing this document, SGX patches have not landed on the Linux kernel
+project and `cloud-hypervisor` 0.8.0 doesn't support SGX, so specific versions of `cloud-hypervisor`,
+guest and host kernel must be installed to enable SGX.
+
+## Install Host kernel with SGX support
+
+The following commands were tested on fedora 32, they might work on other distros too.
+
+```sh
+git clone --depth=1 https://github.com/intel/kvm-sgx
```

Comment:
Can you add a `$ ` prefix to all the shell code lines (here and below) to comply with https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#code-blocks.